backend/src/test/resources/tests/query/UPDATE_IMPORT_TESTS/SIMPLE_TREECONCEPT_Query.test.json

```diff
@@ -0,0 +1,84 @@
+{
+	"type": "QUERY_TEST",
+	"label": "UPDATE_IMPORT_TESTS Test",
+	"expectedCsv": "tests/query/UPDATE_IMPORT_TESTS/expected.csv",
+	"query": {
+		"type": "CONCEPT_QUERY",
+		"root": {
+			"type": "CONCEPT",
+			"ids": [
+				"test_tree.test_child1"
+			],
+			"tables": [
+				{
+					"id": "test_tree.test_column",
+					"filters": []
+				}
+			]
+		}
+	},
+	"concepts": [
+		{
+			"label": "test_tree",
+			"type": "TREE",
+			"connectors": [
+				{
+					"label": "tree_label",
+					"name": "test_column",
+					"column": "table.test_column",
+					"validityDates": {
+						"label": "datum",
+						"column": "table.datum"
+					}
+				}
+			],
+			"children": [
+				{
+					"label": "test_child1",
+					"description": " ",
+					"condition": {
+						"type": "PREFIX_LIST",
+						"prefixes": "A1"
+					},
+					"children": []
+				},
+				{
+					"label": "test_child2",
+					"description": " ",
+					"condition": {
+						"type": "PREFIX_LIST",
+						"prefixes": "B2"
+					},
+					"children": []
+				}
+			]
+		}
+	],
+	"content": {
+		"tables": [
+			{
+				"csv": [
```

Comment:
Ich glaube es wäre sinnvoller, wenn du so etwas machen möchtest, das über key-value zu lösen, damit du die Namen der imports setzen kannst.

Reply:
das Problem ist, wenn ich das so umsetze, dann müsste ich über in den verschiedenen Tests anpassen

Reply:
@Njimefo ich kann dir helfen das per Regex in allen Tests anzupassen

Reply:
Ich frag mich auch, ob es enn wichtig ist, dass in einer test-json zu laden, wenn es nicht wirklich benutzt wird in einem json test. In anderen programmatic tests laden wir die dateien ja einfach.